cmd/minishift/cmd/start.go

```diff
@@ -161,6 +163,12 @@ func runStart(cmd *cobra.Command, args []string) {
 	// we need to determine whether this is a restart prior to potentially creating a new VM
 	isRestart := cmdUtil.VMExists(libMachineClient, constants.MachineName)
```

Comment:
Can we move this one as much as possible to the top of the function?

This file now contains three `isRestart`, but we use them all as `!isRestart`... so maybe we should invert the meaning of the variable to something like `isFirstStart`? WDYT

Reply:
@gbraad I think isRestart is fine and communicates the right intent.

Reply:
On second thought I guess you meant `isFreshStart` I am fine with both isRestart and isFreshStart.

Reply:
The most important is to move it to the top of the function, as this is a conditional statement which is important for the rest of the execution.

Reply:
not a huge fan of inverted logic... especially when this is the only way we use it. inversion can lead to trouble in semantics (reading it and reasoning about it)